### ISSUE TITLE
The filter update dates are not all correct.

### ISSUE THREAD

**Issue by Quorum75** [AdguardTeam/AdguardForMac#1039]
### Steps to reproduce

1.
2.
3.
4.

### Expected behavior
 Getting correct update dates for each filter.

### Actual behavior
 The filter update dates are not all correct.

<details><summary>Screenshot:</summary>

</details>
Incorrect Date:
![Ext List](https://user-images.githubusercontent.com/49731083/153914920-7ea225be-d968-4963-96a3-1705d2e09526.jpg)

Correct Date:
![Pete List](https://user-images.githubusercontent.com/49731083/153914986-c4478579-26e5-44c1-af7f-ac5bb21dd575.jpg)

### Customer ID 
 578450

### Your environment

macOS 12.2.1 (21D62)
AG Desktop  version 2.7.0.1094 release (CL-1.8.276, DNS-1.6.72)

**Comment by Chinaski1** [comment]
Hello there!

As far as I remember we have already fixed this issue in the [Nightly](http://agrd.io/mac) version.
Fix will be available in the release channel soon.
Stay tuned :)

**Comment by ameshkov** [comment]
Well, these are different dates.

The one you see inside the filter list is when it was compiled.
The one you see in the UI is when the filter list was downloaded by the program.

**Comment by Quorum75** [comment]
@ameshkov 

Then, how do you explain that when updating the filters the dates never changed? I am confused. 

Also, only the filters provided by AG get their dates updated, why is that?

As mentioned in my previous message, dates used to appear in AG for iOS. Today, there is no more any date in the UI beside the filter name like it used to be. 

I am using AG for years now and this problem appeared recently. So, something happened. It is annoying because we have no idea if the filters have been updated or not.

**Comment by ameshkov** [comment]
Yeah, I see it now, the bug is about custom filter lists update time.

It does look wrong.

**Comment by planet0** [comment]
I should mention that I think this is the same issue as #1022

**Comment by IaroslavKhvorostenko** [comment]
Hello @Quorum75 
Thank you for the report. You will be notified once a solution has been implemented
